samples/net/sockets/http_get/src/http_get.c

```diff
@@ -25,8 +25,14 @@
 
 #if defined(CONFIG_NET_SOCKETS_SOCKOPT_TLS)
 #include <net/tls_credentials.h>
+#if defined(CONFIG_TLS_CREDENTIAL_FILENAMES)
```

Comment:
IMO we could do all this discrimination in `ca_certificate.h` and just include it as it was. The sample code would not need to be modifed at all, as the only difference is on what we keep in the `ca_certificate` array.

Reply:
True, that would appear less intrusive.